### ISSUE TITLE
Integrate media cards with feed page

### ISSUE THREAD

**Issue by Matteas-Eden** [softeng-701-group-5/softeng-701-assignment-1#59]
**User Story**
As a developer, I'd like to integrate the media cards with the layout of the feed page, so that a coherent design for the webapp can be achieved.

**Acceptance Criteria**

- Media cards added to feed page
- Single column layout that adapts nicely to different platforms (mobile, tablet, desktop)

**Notes**
- Blocked by #41

**Comment by jordansimsmith** [comment]
marked as duplicate of #50, will close as discussed with @Matteas-Eden
